lib/project_types/script/commands/create.rb

```diff
@@ -30,8 +30,8 @@ def call(args, _name)
       end
 
       def self.help
-        allowed_values = "{{cyan:discount}} and {{cyan:unit_limit_per_order}}"
-        ShopifyCli::Context.message('script.create.help', ShopifyCli::TOOL_NAME, allowed_values)
+        allowed_values = Script::Layers::Application::ExtensionPoints.types.map { |type| "{{cyan:#{type}}}" }
```

Comment:
I'd probably add a simple test expecting this to be called (like `Script::Layers::Application::ExtensionPoints.expects(:types).returns(%w(ep1 ep2))`). And maybe ensure the `Context.message` method is called with the properly formatted values. What do you think?

Reply:
That sounds good, I will start that right away!